src/main/java/installer/Installer.java

```diff
@@ -1,4 +1,5 @@
 package installer;
+//TODO: does this need to be a separate package?
```

Comment:
Technically this `Installer` has nothing to do with the application; it's purely for release/devops purpose. It's to create a self-extracting zip file. Hence I separated the package.